pkg/virt-controller/services/renderresources.go

```diff
@@ -265,10 +265,11 @@ func copyResources(srcResources, dstResources k8sv1.ResourceList) {
 // This includes the memory needed for the guest and memory
 // for Qemu and OS overhead.
 //
-// The return value is overhead memory quantity
+// # The return value is overhead memory quantity
```

Comment:
nit `s/#//`

Reply:
It was automatically added after saving the file. I have to remove the empty line above to get ride of the `#`.